velox/common/encode/Base64.cpp

```diff
@@ -428,6 +438,10 @@ std::string Base64::encode_url(const folly::IOBuf* data) {
   return encodeImpl(IOBufWrapper(data), kBase64UrlCharset, false);
 }
 
+static void decode(const char* data, size_t size, char* output) {
```

Comment:
This function doesn't seem to have a body. Is this something left over ? Please delete it if not needed.

Reply:
deleted